src/sentry/tasks/post_process.py

```diff
@@ -254,8 +261,22 @@ def post_process_group(
             if not has_reappeared:  # If true, we added the .UNIGNORED reason already
                 if is_new:
                     add_group_to_inbox(event.group, GroupInboxReason.NEW)
+                    inbox_in.send_robust(
+                        project=event.group.project,
+                        user=None,
+                        group=event.group,
+                        sender="post_process_group",
+                        reason="new_issue",
+                    )
                 elif is_regression:
                     add_group_to_inbox(event.group, GroupInboxReason.REGRESSION)
+                    inbox_in.send_robust(
```

Comment:
Should this just be part of `add_group_to_inbox` and we base `reason` off of whatever `GroupInboxReason` is?